File: RAI/EjemploBanco/JavaImplementation/Vista.java
public class Vista extends Cuenta{
    private int limiteSaldo;

    public Vista(int id, int saldo, int limiteSaldo){
        super(id,saldo);
        this.limiteSaldo = limiteSaldo;
    }

    public int getLimiteSaldo(){
        return this.limiteSaldo;
    }

    public String toString(){
        return super.toString() + "(" + this.limiteSaldo + ")";
    }
}

File: RAI/EjemploBanco/python_implementation/Vista.py
from Cuenta import Cuenta

class Vista(Cuenta):
    def __init__(self, id, saldo, limite_saldo):
        super().__init__(id, saldo)
        self.limite_saldo = limite_saldo

    def get_limite_saldo(self):
        return self.limite_saldo

    def __str__(self):
        return super().__str__() + "(" + str(self.limite_saldo) + ")"
